.github/actions/run-api-tests/script.sh

```diff
@@ -9,7 +9,8 @@ export ENV_PATH="$(pwd)/test-apps/api/.env"
 export JWT_SECRET="aSecret"
 
 opts=($DB_OPTIONS)
+jestOptions=($JEST_OPTIONS)
 
 yarn nx run-many --target=build --nx-ignore-cycles --skip-nx-cache
 yarn run test:generate-app --appPath=test-apps/api "${opts[@]}"
-yarn run test:api --no-generate-app
+yarn run test:api --no-generate-app "${jestOptions[@]}"
```

Comment:
how does our cli interact with the --shard option here?

Reply:
test:api runs "test:api": "node test/scripts/run-api-tests.js", what runs jest so you give it the options for jest it will give them to jest.